src/main/groovy/FrontEndTestJobBuilder.groovy

```diff
@@ -0,0 +1,42 @@
+import JsJobBuilder
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+
+class FrontEndTestJobBuilder {
+
+    String name
+    String description
+    String gitBranch = 'master'
+    String pollScmSchedule = '@daily'
```

Comment:
Currently this field isn't being used, can we add it, either here or in the base?